src/classes/aflightentry.h

```diff
@@ -1,18 +1,33 @@
+/*
+ *openPilot Log - A FOSS Pilot Logbook Application
+ *Copyright (C) 2020  Felix Turowsky
+ *
+ *This program is free software: you can redistribute it and/or modify
+ *it under the terms of the GNU General Public License as published by
+ *the Free Software Foundation, either version 3 of the License, or
+ *(at your option) any later version.
+ *
+ *This program is distributed in the hope that it will be useful,
+ *but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *GNU General Public License for more details.
+ *
+ *You should have received a copy of the GNU General Public License
+ *along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ */
 #ifndef AFLIGHTENTRY_H
 #define AFLIGHTENTRY_H
 
-#include "src/experimental/aentry.h"
+#include "src/classes/aentry.h"
 #include "src/functions/acalc.h"
 
-namespace experimental {
-
 class AFlightEntry : public AEntry {
 public:
     AFlightEntry();
     AFlightEntry(const AFlightEntry& pe) = default;
     AFlightEntry& operator=(const AFlightEntry& pe) = default;
     AFlightEntry(int row_id);
-    AFlightEntry(TableData table_data);
+    AFlightEntry(RowData table_data);
 
     enum pilot { pic, sic, thirdPilot };
```

Comment:
perhaps rename to `pilotPosition` or some aviation specific name for what the pilot is.
Just pilot feels a bit ambiguous. Especially for someone not in the industry!